retrofit/src/main/java/retrofit/Retrofit.java

```diff
@@ -261,6 +265,12 @@ public Builder validateEagerly() {
       return this;
     }
 
+    /** Configure debug logging mechanism. */
+    public Builder setLogger(Logger logger) {
```

Comment:
Other methods of the `Builder` don't have `set` prefix, I'd remove it for consistency :)